server/core/src/https/v1.rs

```diff
@@ -971,6 +971,19 @@ pub async fn account_delete_id_unix_credential(
     to_axum_response(res)
 }
 
+pub async fn identify_user(
```

Comment:
A more verbose fn name here would be good IMO. Note that (most) other fn's define the http verb type so this could be something like "person_post_identify_user".